tests/lib/rules/jsx-no-literals.js

```diff
@@ -415,33 +427,33 @@ ruleTester.run('jsx-no-literals', rule, {
       errors: [{message: stringsMessage('`Test`')}]
     }, {
       code: '<Foo bar={`Test`} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
       errors: [{message: stringsMessage('`Test`')}]
     }, {
       code: '<Foo bar={`${baz}`} />',
-      options: [{noStrings: true}],
+      options: [{noStrings: true, validateProps: true}],
```

Comment:
These test changes show that the option (that doesn't exist yet) seems to already be `true` by default prior to this PR. In that case, this option should probably be more like `ignoreProps: true` which defaults to false.

Reply:
@ljharb just checking... Is this the only requested change pending?

Reply:
@Gasparila at the moment, yes - i'll have to give it a rereview after it's next updated

Reply:
Just found this great react eslint rule to use in a big product and faced the same problem that it goes through the props, which was a bit surprising, but ok.
In short, the current rule behavior is to check props, facilitating it to be disabled by default may break some existing users functionalities, who may rely on this.
+1 , is should be either changed to ````validateProps`(`true` default)``` in the current naming, or switched to `ignoreProps`

Reply:
Hey @ljharb @SalimBensiali
I created a PR to switch to `ignoreProps: true` and adjusted the code, docs and tests to pass: https://github.com/iiison/eslint-plugin-react/pull/1
It targets the current branch, can you please review when you have time, and let me know if you'd need me to merge it, or you'd prefer other way of having this arrived in the main repo